SignalServiceKit/src/Devices/OWSRecordTranscriptJob.m

```diff
@@ -79,24 +80,53 @@ - (void)runWithAttachmentHandler:(void (^)(TSAttachmentStream *attachmentStream)
 
     OWSAttachmentsProcessor *attachmentsProcessor =
         [[OWSAttachmentsProcessor alloc] initWithAttachmentProtos:transcript.attachmentPointerProtos
-                                                        timestamp:transcript.timestamp
                                                             relay:transcript.relay
-                                                           thread:thread
                                                    networkManager:self.networkManager
-                                                   primaryStorage:self.primaryStorage
                                                       transaction:transaction];
 
+
     // TODO group updates. Currently desktop doesn't support group updates, so not a problem yet.
     TSOutgoingMessage *outgoingMessage =
         [[TSOutgoingMessage alloc] initOutgoingMessageWithTimestamp:transcript.timestamp
-                                                           inThread:thread
+                                                           inThread:transcript.thread
                                                         messageBody:transcript.body
                                                       attachmentIds:[attachmentsProcessor.attachmentIds mutableCopy]
                                                    expiresInSeconds:transcript.expirationDuration
                                                     expireStartedAt:transcript.expirationStartedAt
                                                      isVoiceMessage:NO
                                                    groupMetaMessage:TSGroupMessageNone
-                                                      quotedMessage:nil];
+                                                      quotedMessage:transcript.quotedMessage];
+
+    TSQuotedMessage *_Nullable quotedMessage = transcript.quotedMessage;
```

Comment:
We need to retest this once Desktop can actually send quoted replies. I'm going to add a canaray: `OWSAssert(transcript.quotedMessage == nil)`

Reply:
Done.